app/_includes/service-catalog/service-catalog-discovery.html

```diff
@@ -0,0 +1,2 @@
+* Automatic resource discovery {% if include.discovery_support == true %}is{% else %}isn't{% endif %} supported by this integration
```

Comment:
I don't know if this should be a concern, but conditions within a sentence can make things more difficult for translation